src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/TagletWriterImpl.java

```diff
@@ -504,7 +504,7 @@ public Content throwsTagOutput(Element element, ThrowsTree throwsTag, TypeMirror
            excName = htmlWriter.getLink(new HtmlLinkInfo(configuration, HtmlLinkInfo.Kind.MEMBER,
                    substituteType));
         } else if (exception == null) {
-            excName = new RawHtml(ch.getExceptionName(throwsTag).toString());
+            excName = new RawHtml(throwsTag.getExceptionName().toString());
```

Comment:
It's not your code or edit here, but I'll mention it anyway ...

We should be on the lookout for unnecessary use of `RawHtml`.  It looks like this is a candidate for a (later) cleanup to use just `Text`, not `RawHtml`.